agent/agent.go

```diff
@@ -712,6 +728,7 @@ func (a *Agent) StopInputPlugin(input string) {
 	}
 
 	statusChannel <- Stop
```

Comment:
Weren't we going to move away from the start/stop thing and just move to a generic ```struct{}``` channel? Why do we still need to be passing in strings?

Reply:
oops will change